lib/internal/errors.js

```diff
@@ -1411,11 +1411,11 @@ E('ERR_TLS_CERT_ALTNAME_INVALID', function(reason, host, cert) {
 }, Error);
 E('ERR_TLS_DH_PARAM_SIZE', 'DH parameter size %s is less than 2048', Error);
 E('ERR_TLS_HANDSHAKE_TIMEOUT', 'TLS handshake timeout', Error);
-E('ERR_TLS_INVALID_CONTEXT', '%s must be a SecureContext', TypeError),
-E('ERR_TLS_INVALID_STATE', 'TLS socket connection must be securely established',
-  Error),
+E('ERR_TLS_INVALID_CONTEXT', '%s must be a SecureContext', TypeError);
 E('ERR_TLS_INVALID_PROTOCOL_VERSION',
   '%j is not a valid %s TLS protocol version', TypeError);
+E('ERR_TLS_INVALID_STATE', 'TLS socket connection must be securely established',
+  Error);
```

Comment:
This seems unrelated.

Reply:
It's not. There was a comma operator before, instead of a semicolon